### PR TITLE
gingerblue: update port description and dependency on XML::Parser

### PR DESCRIPTION
**Pull request by oleaamot**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.0.1 21A559 arm64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: gnome/gingerblue/Portfile [hunk 1]
```diff
@@ -10,8 +10,8 @@ platforms           darwin
 license             GPL-3+
 maintainers         {gnome.org:ole @oleaamot} \
                     openmaintainer
-description         GNOME Gingerblue
-long_description    Locate Internet Radio Stations
+description         GNOME Gingerblue is Free Music Software for GTK+/GNOME.
+long_description    It supports immediate audio recording in compressed Ogg encoded audio files stored in the ~/Music/ directory with XSPF 1.0 playlist from the line input on a computer or remote audio cards through USB connection through PipeWire with GStreamer.
```
Comment (pmetzger) [inline gnome/gingerblue/Portfile:14]: Could this be wrapped to 80 columns?

**Review by pmetzger — CHANGES_REQUESTED** [review]

File: gnome/gingerblue/Portfile (hunk 1 above)
Comment (oleaamot) [reply gnome/gingerblue/Portfile:14]: The long_description can't be wrapped.

Comment (pmetzger) [reply gnome/gingerblue/Portfile:14]: Every other port does it. Look at how another port handles this.

Comment (oleaamot) [reply gnome/gingerblue/Portfile]: Fixed in the latest commit.

**Comment by oleaamot** [comment]
Fixed in the latest commit in https://github.com/oleaamot/macports-ports/commit/d8c0ff50686b801439911c7c324b01df3821162f

**Review by mascguy — APPROVED** [review]

File: gnome/gingerblue/Portfile [hunk 2]
```diff
@@ -10,8 +12,13 @@ platforms           darwin
 license             GPL-3+
 maintainers         {gnome.org:ole @oleaamot} \
                     openmaintainer
-description         GNOME Gingerblue
-long_description    Locate Internet Radio Stations
+description         GNOME Gingerblue is Free Music Software for GTK+/GNOME.
+long_description    {*}${description} \
```
Comment (mascguy) [inline gnome/gingerblue/Portfile:16]: Ole, note that the `long_description` should also start with the info from `description`.

And while that can be copy-pasted from the `description`, it's better to simply include it as I did here.

Comment (mascguy) [reply gnome/gingerblue/Portfile:16]: Also, be careful not to use tabs anywhere. I fixed them for you already though, so this PR is all set.

**Review by mascguy — APPROVED** [review]

**Comment by oleaamot** [comment]
Thanks!

**Comment by mascguy** [comment]
Tested locally, and looks good. Merging.

**Pull request merged**